sbt-gcs-plugin/src/main/scala/org/latestbit/sbt/gcs/artifactregistry/GcsArtifactRegistryUrlConnection.scala

```diff
@@ -68,7 +58,7 @@ class GcsArtifactRegistryUrlConnection( googleHttpRequestFactory: HttpRequestFac
         }
         if (responseCode < 400) {
```

Comment:
I think we need to remove this check since it is not valid anymore. This kind of works now since responseCode is -1 (default value), but it looks now like accidental side effect :)

Reply:
Absolutely true. I push a fix that removes the if statement.